rosbag2_interfaces/srv/Pause.srv

```diff
@@ -0,0 +1 @@
+---
```

Comment:
http://docs.ros.org/en/api/std_srvs/html/srv/Empty.html

Reply:
These are more accurately `Trigger`s, in terms of what they do - I just couldn't think of a failure case at the time. After doing some integration - I'm thinking that if the player is constructed, but not yet started, that could return `success=false`